pandas/tseries/period.py

```diff
@@ -777,6 +777,15 @@ def get_indexer(self, target, method=None, limit=None, tolerance=None):
         return Index.get_indexer(self._int64index, target, method,
                                  limit, tolerance)
 
+    def _get_unique_index(self, dropna=False):
+        """
+        wrap Index._get_unique_index to handle NaT
+        """
+        res = super(PeriodIndex, self)._get_unique_index(dropna=dropna)
```

Comment:
This is needed to handle ``NaT``. #13979 and #13984 makes ``_get_unique_index`` itself unnecessary.